.config/dotnet-tools.json

```diff
@@ -25,6 +25,12 @@
       "commands": [
         "generate-sbom"
       ]
-    }
+    },
+    "Google.Api.Generator": {
+      "version": "1.5.0",
```

Comment:
We had 1.4.32 before...

Reply:
Yup, I figured that we'd make the first release as a dotnet tool 1.5.0. But that's very much up for debate :)

Reply:
OK, fine by me, just in case it was left over from you trying things locally.